src/config/feature-flags.ts

```diff
@@ -90,3 +91,7 @@ export const isBlocked = async (installationId: number, logger: Logger): Promise
 		return false;
 	}
 };
+
+export const shouldTagBackfillRequests = async (): Promise<boolean> => {
+	return booleanFlag(BooleanFlags.TAG_BACKFILL_REQUESTS, false);
+};
```

Comment:
I know you don't usually expose ffs as functions, but I thought this was a bit easier to mock on tests

Reply:
We don't want to have to maintain multiple functions.  Current way to mock is a pretty simple single line.  Plus, wouldn't you want to pass in the `jiraHost` for this?